lib/grandpa/grandpa.go

```diff
@@ -41,16 +41,19 @@ type Service struct {
 	stopped    bool
 
 	// current state information
-	state           *State                             // current state
-	prevotes        map[ed25519.PublicKeyBytes]*Vote   // pre-votes for next state
-	precommits      map[ed25519.PublicKeyBytes]*Vote   // pre-commits for next state
-	pvEquivocations map[ed25519.PublicKeyBytes][]*Vote // equivocatory votes for current pre-vote stage
-	pcEquivocations map[ed25519.PublicKeyBytes][]*Vote // equivocatory votes for current pre-commit stage
-	head            *types.Header                      // most recently finalized block
+	state            *State                             // current state
+	prevotes         map[ed25519.PublicKeyBytes]*Vote   // pre-votes for the current round
+	precommits       map[ed25519.PublicKeyBytes]*Vote   // pre-commits for the current round
+	pvJustifications []*Justification                   // pre-vote justifications for the current round TODO: is this used anywhere?
```

Comment:
`pv` and `pc` are a it hard to read and distinguish, nor are they well-known abbreviations, fine to use shortened versions within methods but it might be nice to use the full `prevoteJustification` and `precommitJustification` or for the grandpa service struct or something a bit easier to read and distinguish.